include/inexor/vulkan-renderer/frame_graph.hpp

```diff
@@ -242,8 +242,8 @@ class PhysicalResource : public FrameGraphObject {
 
 protected:
     // TODO: Add OOP device functions (see above todo) and only store a wrapper::Device here.
-    VmaAllocator m_allocator;
-    VkDevice m_device;
+    VmaAllocator m_allocator{VK_NULL_HANDLE};
+    VkDevice m_device{VK_NULL_HANDLE};
```

Comment:
Don't think this is needed since they're always initialised by the constructor.

Reply:
The problem is that if we add or change the constructor and they do not hurt or?

Reply:
Clang-tidy should give a warning in that case, but in this case (and the format one below) these members should actually be `const` so that they have to be initialised by the constructor.